src/tests/BuildWasmApps/testassets/Blazor_net50/wwwroot/css/open-iconic/FONT-LICENSE

```diff
@@ -0,0 +1,86 @@
+SIL OPEN FONT LICENSE Version 1.1
```

Comment:
I'd really like to avoid checking in bootstrap/fonts/css etc. assets into the repo. Can we make a minimal test instead?

Reply:
This is the blazor template app, but I guess since we aren't running the app, we could drop these.